save_result.json

```diff
@@ -1,15 +1,15 @@
 {
     "id": "save_result",
-    "summary": "Save processed data to storage",
-    "description": "Saves processed data to the server-side user workspace of the authenticated user. This process aims to be compatible with GDAL/OGR formats and options. STAC-compatible metadata should be stored with the processed data.\n\nCalling this process may be rejected by back-ends in the context of secondary web services.",
+    "summary": "Save processed data",
+    "description": "Makes the processed data available in the given file format to the corresponding medium that is relevant for the context this processes is applied in:\n\n* For **batch jobs** the data is stored in the server-side user workspace of the authenticated user. STAC-compatible metadata are usually made available with the processed data.\n* For **synchronous processing** the data is directly sent to the client.\n* For secondary web services the behavior depends on the service type. Usually, the data is directly provided to the secondary web service, but calling this process may also be rejected by some service types.",
     "categories": [
         "cubes",
         "export"
     ],
     "parameters": [
         {
             "name": "data",
-            "description": "The data to save.",
+            "description": "The data to make available in the given file format.",
```

Comment:
just an idea: "deliver" instead of "make available". Works both for synchronous and  batch in my head

Reply:
Works for me, changed.